types/tx.go

```diff
@@ -118,3 +120,18 @@ type TxResult struct {
 	Tx     Tx                     `json:"tx"`
 	Result abci.ResponseDeliverTx `json:"result"`
 }
+
+// ComputeAminoOverhead calculates the overhead for amino encoding a transaction.
+// The overhead consists of varint encoding the field number and the wire type
+// (= length-delimited = 2), and another varint encoding the length of the
```

Comment:
What does `(= length-delimited = 2)` mean?

Reply:
The wire-type is what we call `Typ3_ByteLength` in amino, or, length-delimited in proto3, or, equal to `2` in both cases. Maybe that comment is too verbose?

Reply:
Ah I see. Yeh I would have been more explicit, like `(2, in this case, for a length-delimited type)` but that's fine